src/main/res/values/strings.xml

```diff
@@ -216,6 +216,21 @@ limitations under the License.
     <string name="export_progress_message">Exporting to %1$s&#8230;</string>
     <string name="export_success">Exported %1$s to %2$s</string>
     <string name="export_title">Export to&#8230;</string>
+    <string name="export_track_already_exists_msg">The track %1$s already exists at the destination directory!</string>
+    <string name="export_all_end_with_errors">Completed with %1$d errors</string>
+    <string name="export_all_end_with_error">Completed with an error</string>
```

Comment:
sed s/an/one/

Reply:
Should be done with plurals:

```
    <plurals name="export_all_end_with_errors">
        <item quantity="one">Completed with one error</item>
        <item quantity="other">Completed with %1$d errors</item>
    </plurals>
```

And the usage:
```
String msg = getResources().getQuantityString(R.plurals.export_all_end_with_errors, trackExportErrorCount, trackExportErrorCount);

```